src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -540,20 +540,38 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                 }
             }
 
-            ReadProjectConfigurationDescription(e.BuildEventContext, e.Items);
+            var projectKey = (e.BuildEventContext.NodeId, e.BuildEventContext.ProjectContextId);
```

Comment:
Can e.BuildEventContext be null here?

Reply:
No, BuildEventContext should always be set